src/app/components/cds/inputs/TextField.test.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import VolumeUpIcon from '@material-ui/icons/VolumeUp';
-import VolumeOffIcon from '@material-ui/icons/VolumeOff';
 import { FormattedMessage } from 'react-intl';
 import TextField from './TextField';
+import VolumeUpIcon from '../../../icons/volume_up.svg';
```

Comment:
@dariusk running into another testing issue after this change. I did some googling, but got lost in my less than average jest knowledge. Do you know of a change to make to help tests like this?

Reply:
Looking at these test failures now! I'll let you know.